fleetspeak/src/client/daemonservice/execution/execution_test.go

```diff
@@ -172,10 +176,11 @@ func TestStd(t *testing.T) {
 	}
 
 	// Flushes should happen when the buffer is full. The last flush might take up
-	// to outFlushTime to occur, the rest should occur quickly.
+	// to 5 seconds occur, the rest should occur quickly.
 	runtime := time.Since(start)
-	if runtime > 2*outFlushTime {
-		t.Errorf("took %v to receive std data, should be less than %v", runtime, 2*outFlushTime)
+	deadline := 10 * time.Second
```

Comment:
Shouldn't this be moved to the `const` section at the top?

Reply:
We haven't been bothering with that in test files, and in this case it just need to be consistent with the 5 second flush time set in this test method.

Reply:
Oh, I haven't noticed that this is a test code. In this case that is totally fine (even better).